frontend/app/src/main/java/com/quickpick/CreateNewListActivity.java

```diff
@@ -1,14 +1,43 @@
 package com.quickpick;
 
-import androidx.appcompat.app.AppCompatActivity;
-
 import android.os.Bundle;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.app.AppCompatActivity;
 
 public class CreateNewListActivity extends AppCompatActivity {
 
+    private Button addEntries;
+
+    private Button createList;
+
+    private EditText editName;
+
+    private EditText editDescription;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_create_new_list);
+
+        editName = (EditText) findViewById(R.id.create_list_name_exit_text);
```

Comment:
Not sure if you're using them in the future, but avoid making these into private instance variables unless they're used in multiple methods

Reply:
added a todo to make note of this for future commits